suricata-ocsf/package.yaml

```
@@ -247,8 +247,13 @@ pipelines:
         port: suricata.src_port,
       }
       drop suricata.src_ip, suricata.src_port
-      ocsf.activity_id = $activity_id[suricata.smb.disposition].otherwise(99)
-      ocsf.activity_name = $activity_name[suricata.smb.disposition].otherwise(suricata.smb.command)
+      if suricata.smb.disposition != null {
+        ocsf.activity_id = $activity_id[suricata.smb.disposition].otherwise(99)
```

```suggestion
        ocsf.activity_id = $activity_id.get(suricata.smb.disposition, 99)
```

Should we replace `otherwise` with this pattern? 👆